molgenis-data-migrate/src/main/java/org/molgenis/data/migrate/version/Step42ReplaceFormsUrlRow.java

```diff
@@ -0,0 +1,40 @@
+package org.molgenis.data.migrate.version;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import javax.sql.DataSource;
+import org.molgenis.data.migrate.framework.MolgenisUpgrade;
+import org.molgenis.util.ResourceUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+public class Step42ReplaceFormsUrlRow extends MolgenisUpgrade {
+  private static final Logger LOG = LoggerFactory.getLogger(Step42ReplaceFormsUrlRow.class);
+  private final DataSource dataSource;
+
+  public Step42ReplaceFormsUrlRow(DataSource dataSource) {
+    super(41, 42);
```

Comment:
You still need to update the version in https://github.com/molgenis/molgenis/blob/0e4aef3150f32f3a74e350641f95e8faa2e2dd4b/molgenis-data-migrate/src/main/java/org/molgenis/data/migrate/version/MolgenisVersionService.java#L20

Reply:
done